source/includes/fundamentals/code-examples/ExtendedJson.cs

```diff
@@ -14,6 +14,19 @@ public static void Main(string[] args)
             Console.WriteLine(document.ToJson());
             // end-read-ejson
         }
+        
+        {
+            // start-read-ejson-reader
+            var ejson = "{\n\"_id\": { \"$oid\": \"573a1391f29313caabcd9637\" },\n  \"createdAt\": { \"$date\": { \"$numberLong\": \"1601499609\" }},\n\"numViews\": { \"$numberLong\": \"36520312\" }\n}\n\n";
+            var subject = new BsonDocumentSerializer();
+            using (var reader = new JsonReader(ejson))
+            {
+                var context = BsonDeserializationContext.CreateRoot(reader);
+                var document = subject.Deserialize<BsonDocument>(context);
+                Console.WriteLine(document.ToJson());
+            }
+            // end-read-ejson-reader
+        }
```

Comment:
There's  a much easier way to parse JSON into a `BsonDocument`:

    var document = BsonDocument.Parse(json);

Reply:
I'm not sure what the point of this example is. It's just a harder way to do something that can be done more simply.